WordPress/src/main/res/layout/reader_parent_post_list_fragment.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <com.google.android.material.tabs.TabLayout
+        android:id="@+id/tab_layout"
```

Comment:
I think there is a flag that stops the tabs from having constant width, now they look like this:
![Screenshot_1588852773](https://user-images.githubusercontent.com/1079756/81291917-fdf75780-906a-11ea-9883-f0b4459a4fde.png)

Reply:
Good catch! Fixed in f8338b915a0e8b0d547bf24c8cd7f67e5ebe4178